storage/storage_image.go

```diff
@@ -761,10 +761,11 @@ func (s *storageImageDestination) queueOrCommit(ctx context.Context, blob types.
 	}
 
 	for info := s.indexToPulledBlob[index]; info != nil; info = s.indexToPulledBlob[index] {
+		info := s.indexToPulledBlob[index]
```

Comment:
When does the make a difference? I can’t think of a case.

Reply:
It doesn't make a difference.  Missed that line when cleaning up all the debugging statements.  Thanks for catching!